content/applications/sales/sales/send_quotations/orders_and_variants.rst

```diff
@@ -1,79 +1,123 @@
+product variants can be viewed, modified, and added.
 
-After saving the settings, configure the product form. Head over to :menuselection:`Sales -->
-Products --> Products` and select any product (or click :guilabel:`Create` to create a new one).
-On the product form, click :guilabel:`Edit`, and then click on the :guilabel:`Variants` tab. Under
-:guilabel:`Sales Variant Selection`, choose either :guilabel:`Product Configurator` or
-:guilabel:`Order Grid Entry`. This selection determines which method is used to add the product to
-quotations or sales orders. Finally, click :guilabel:`Save` to apply the setting.
+At the bottom of the :guilabel:`Attributes & Variants` tab, there is a :guilabel:`Sales Variant
+Selection` section with two options: :guilabel:`Product Configurator` and :guilabel:`Order Grid
+Entry`.
 
-.. image:: orders_and_variants/sales-variant-selection.png
+.. note::
+   It should be noted that these options **only** appear if at least two values of an attribute have
+   been added to the record.
+
+.. image:: orders_and_variants/attributes-variants-tab-selection-options.png
    :align: center
-   :alt: Sales variant selection.
+   :alt: Sales variant selection options on the attributes and variants tab on product form.
+
+These options determine which method is used when adding product variants to quotations or sales
+orders.
+
+The :guilabel:`Product Configurator` provides a pop-up window that neatly displays all the available
+product variants for that particular product when it's added to a quotation. However, only one
+variant can be selected/added at a time.
+
+The :guilabel:`Order Grid Entry` provides the same information as the :guilabel:`Product
+Configurator` in a table layout, allowing the user to select larger numbers of unique product
+variants, and add them to a quotation/sales order, in a single view.
 
 .. note::
-   The product must have at least two variants (one attribute with two values) created in order for
-   the :guilabel:`Sales Variant Selection` option to appear. Click :guilabel:`Add a line` under
+   The product must also have at least two variants (one attribute with two values) created in order
+   for the :guilabel:`Sales Variant Selection` option to appear. Click :guilabel:`Add a line` under
    :guilabel:`Attribute` to add attributes and values to this product and create product variants.
 
-Use the Product Configurator
-============================
+Product configurator
+====================
 
-The :guilabel:`Product Configurator` appears on a quotation or sales order when products are added
-that have both variants configured and the :guilabel:`Product Configurator` option selected.
+The :guilabel:`Product Configurator` feature appears as a :guilabel:`Configure` pop-up window, as
+soon as a product with (at least two) variants is added to a quotation or sales order, but **only**
+if the :guilabel:`Product Configurator` option is selected on its product form.
```

Comment:
When referring to the feature no guilabel is needed, but when referring to the option field on the UI guilabel is needed
```suggestion
The product configurator feature appears as a :guilabel:`Configure` pop-up window, as soon as a
product with (at least two) variants is added to a quotation or sales order, but **only** if the
:guilabel:`Product Configurator` option is selected on its product form.
```